src/phonebook/PhoneBook.java

```diff
@@ -0,0 +1,51 @@
+package phonebook;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PhoneBook {
+    private List<Record> records;
+
+    public PhoneBook() {
+        this.records = new ArrayList<>();
+    }
+
+    public boolean add(Record record) {
+        //Primitive implementation
+        for (Record contact : records) {
+            if (contact.getPhone().equals(record.getPhone())) {
+                System.out.println("This number(" + record.getPhone() + ") is already in use, so it cannot be added.");
```

Comment:
One contact (name) can have different numbers => multiple records, so this check is redundant